docs/sorbet.md

```diff
@@ -6,6 +6,19 @@ Mutant aims to implement full sorbet support. Full support includes:
 * Being able to cover mutations via the type system. This capability is being
   developed right now.
 
+For now, all of the mutations to `T` methods are skipped. To better leverage `mutant`'s capabilities
+use local variables or parameterless methods when possible, for example:
```

Comment:
I think we should mention that we are making explicit visitors to evaluate arguments to well known sorbet APIs.

Reply:
I'm sorry, didn't get this.

Reply:
Lets remove this recommendation, I think its diluting the value of mutant, as it adds redundancy. We'll add an explicit visitor to the arguments of `T.must` soon. But asking people to refactor code to something I'd flag (Redundant lvar) is a bad idea.

Reply:
Ok I get it now, I don't know what a visitor is but I think i get the idea. Removing this!